indico/cli/event.py

```diff
@@ -85,3 +87,39 @@ def import_(source_file, create_users, force, verbose, yes, category_id):
         sys.exit(1)
     db.session.commit()
     click.secho(event.external_url, fg='green', bold=True)
+
+
+@cli.command('create-series')
+@click.option('--no-title-sequence', is_flag=True,
+              help='Whether to show the series sequence in the event titles (lectures only)')
+@click.option('--no-links', is_flag=True,
+              help='Whether to show links to other events in the series on the event page')
+@click.argument('event_ids', nargs=-1, type=int)
```

Comment:
I think adding `metavar='EVENT_ID'` or `metavar='EVENT_IDS'` will make the command's `--help` a bit nicer

Reply:
I check some common tools and they use `ARG...` so I did that. I added a screenshot of the `--help` output to the PR

Reply:
yeah, `EVENT_ID...` looks good :)